Makefile

```diff
@@ -1,24 +1,28 @@
 TARGET_OS = linux
 TARGET_ARCH = amd64
-RELEASE_REPO ?= $(GOPATH)/src/github.com/adhocteam/coverage-validator-release
-RAWNPPESCSV ?= npidata_20050523-20171112.csv
-
+RELEASE_DIR = /tmp/coverage-validator-release
 SOURCES = index.html docs.html index_schema.json providers_schema.json plans_schema.json drugs_schema.json static npis.csv Procfile
+NPI_URL:=
```

Comment:
I'm a little lost as to what this is doing - is it setting NPI_URL to a null string?

Reply:
You set it when you call the make target so that we aren't putting the URL in the code.

Reply:
It's just the location of the npi dump file in s3